ironfish/src/mining/stratum/stratumServer.ts

```diff
@@ -171,36 +165,28 @@ export class StratumServer {
           const body = await YupUtils.tryValidate(MiningSubscribeSchema, header.result.body)
 
           if (body.error) {
-            this.addBadClient(client)
-            continue
-            // throw new ClientMessageMalformedError(client, body.error, header.result.method)
+            this.peers.ban(client)
```

Comment:
This introduces an interesting edge-case. Let's say we implement some change, including a version bump. If it leads to a validation failure, we will ban them for the max amount, since this occurs before the version check below this, which only bans for 15 minutes. Same edge case exists for the ban on L157.

Reply:
I think that's an edge case with the handshake message. The developer needs to make all changes to the handshake message subscribe backwards compatible.